### PR TITLE
feat: add and config ESLint, Prettier and EditorConfig

### PR DESCRIPTION
**Pull request by eugustavo**
ESLint and Prettier were added for code standardization. And the EditorConfig to standardize the configuration.

### CODE REVIEW

File: packages/web/package.json [hunk 1]
```diff
@@ -35,7 +35,8 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "lint": "eslint --fix src --ext .ts,.tsx"
```
Comment (diego3g) [inline packages/web/package.json:39]: I think we can remove the `fix` flag here so we have a lint command without fixing to run on the CI.

File: packages/web/src/App.tsx [hunk 2]
```diff
@@ -12,6 +12,6 @@ const App: React.FC = () => {
       </div>
     </Router>
   );
-}
+};
```
Comment (diego3g) [inline packages/web/src/App.tsx:15]: 🤔 Do we need the semicolon here?

File: packages/web/src/components/PaginatedTable/index.tsx [hunk 3]
```diff
@@ -10,35 +10,52 @@ interface Props {
...
 
   return (
     <>
-      { data && (
+      {data && (
         <Pagination>
           <p>
-            <span>{response?.headers['x-total-count']} registro(s)</span>
-            <span>{response?.headers['x-total-page']} página(s)</span>
+            <span>{response?.headers['x-total-count']}
```
Comment (diego3g) [inline packages/web/src/components/PaginatedTable/index.tsx:27]: The identation here got a little messed up. Do you think that we can fix that or ESLint is blocking?

File: packages/web/src/pages/MessageDetail/index.tsx [hunk 4]
```diff
@@ -50,17 +50,17 @@ const MessageForm: React.FC<Props> = ({ match }) => {
...
 
   const handleSendMessage = useCallback(async () => {
     try {
       const response = await axios.post(`/messages/${id}/send`);
 
-      mutate(requestKey, response, false); 
+      mutate(requestKey, response, false);
 
-      console.log('mutate called')
+      console.log('mutate called');
```
Comment (diego3g) [inline packages/web/src/pages/MessageDetail/index.tsx:63]: I think we can remove this `console.log` and any others that you find in the code.

File: packages/web/package.json (hunk 1 above)
Comment (eugustavo) [reply packages/web/package.json:39]: Ok, I will remove.

File: packages/web/src/components/PaginatedTable/index.tsx (hunk 3 above)
Comment (eugustavo) [reply packages/web/src/components/PaginatedTable/index.tsx:27]: An idea that came to my mind is maybe we could move  `response?.headers['x-total-count']` to a `numberOfRegister` variable and `response?.headers['x-total-page']` to `numberOfPages`.

File: packages/web/src/pages/MessageDetail/index.tsx (hunk 4 above)
Comment (eugustavo) [reply packages/web/src/pages/MessageDetail/index.tsx:63]: Ok, I will remove all `console.log`

File: packages/web/src/App.tsx (hunk 2 above)
Comment (eugustavo) [reply packages/web/src/App.tsx:15]: It is not necessary, I can withdraw.

File: packages/web/src/components/PaginatedTable/index.tsx (hunk 3 above)
Comment (diego3g) [reply packages/web/src/components/PaginatedTable/index.tsx:27]: @eugustavo That would be great!

File: packages/web/package.json [hunk 5]
```diff
@@ -35,8 +35,7 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject",
-    "lint": "eslint --fix src --ext .ts,.tsx"
```
Comment (diego3g) [inline packages/web/package.json]: I didn't mean to remove the full line, only the `--fix` option. You can keep the link script.

File: packages/web/src/components/PaginatedTable/index.tsx [hunk 6]
```diff
@@ -19,19 +19,28 @@ const PaginatedTable: React.FC<Props> = ({ request, children }) => {
     loadNext,
   } = request;
 
+  const numberOfRegister = useMemo(() => response?.headers['x-total-count'], [
```
Comment (diego3g) [inline packages/web/src/components/PaginatedTable/index.tsx:22]: `numberOfRegisters` in plural

**Pull request merged**